src/lib.rs

```diff
@@ -69,9 +70,15 @@ pub async fn install_holo_hosted_happs(
     // iterate through the vec and
     // Call http://localhost/holochain-api/install_hosted_happ
     // for each WrappedHeaderHash to install the hosted_happ
-    for (happ_id, bundle_url) in happs {
+    for (happ_id, bundle_url, is_paused) in happs {
         if active_happs.contains(&format!("{:?}", happ_id)) {
             info!("App {:?} already installed", happ_id);
+            if is_paused.to_owned() {
+                info!("Pausing {:?}", happ_id);
+                admin_websocket
+                    .deactivate_app(&happ_id.0.to_string())
+                    .await?;
+            }
```

Comment:
Don't we need to prevent users from using identified instances somehow? This only deactivates the anonymous instance. Does that even prevent users from signing up? I think they can clone the anonymous instance without it being enabled

Reply:
Here I was thinking that envoy should be the one who blocks the connection since it's making a call to HHA and knows that happ status anyway. 

I think the deactivation and uninstalling here on holochain is just to stop the holoport-resources from being used by stopping gossip and clearing up disk space based on the happy status.

Reply:
Okay, but make sure that that work in Envoy is tracked somewhere. I assumed it would be part of this story